lorawan-encoding/src/creator.rs

```diff
@@ -129,6 +133,9 @@ impl<D: AsMut<[u8]>, F: CryptoFactory + Default> JoinAcceptCreator<D, F> {
             return Err(Error::InvalidChannelList);
         }
         let d = self.data.as_mut();
+        if d.len() < 15 + ch_list.len() * 3 {
```

Comment:
`CF_LIST_LEN`? (Though in our case it's defined as 16).

Reply:
you are right, it is always 16 bytes (both type 0 and 1). It is possible to pass in something shorter due to `AsRef<[Frequency<'a>]>` but that is another issue